src/main/java/com/google/cloud/spanner/jdbc/JdbcTypeConverter.java

```diff
@@ -190,39 +190,13 @@ static Object convert(Object value, Type type, Class<?> targetType) throws SQLEx
       if (targetType.isArray() && type.getCode() == Code.ARRAY) {
         if (type.getArrayElementType().getCode() == Code.PROTO
             || type.getArrayElementType().getCode() == Code.BYTES) {
-          Class<?> componentType = targetType.getComponentType();
-          if (AbstractMessage.class.isAssignableFrom(componentType)) {
-            byte[][] result = (byte[][]) ((JdbcArray) value).getArray();
-            Object obj = java.lang.reflect.Array.newInstance(componentType, result.length);
-            for (int i = 0; i < result.length; i++) {
-              if (result[i] != null) {
-                Method parseMethodParseFrom = componentType.getMethod("parseFrom", byte[].class);
-                java.lang.reflect.Array.set(
-                    obj, i, componentType.cast(parseMethodParseFrom.invoke(null, result[i])));
-              }
-            }
-            return obj;
-          }
+          Object res = convertArrayOfProtoMessage(value, targetType);
+          if (res != null) return res;
```

Comment:
nit: use curly braces for all if statements

Reply:
Done